src/sources/dataModel/taxonomic-service.json

```diff
@@ -338,12 +690,53 @@
                     "examples": [
                         "https://creativecommons.org/public-domain/cc0/"
                     ]
+                },
+                "schema:additionalProperty": {
```

Comment:
You can extract this part and $ref it as you use it in multiple places

Reply:
True, but then it does not have the names and descriptions of the different properties we are nesting right? It does not change anything about the object itself, but I kinda like the idea of having an idea where the properties are stored (at which index).